src/model/operation/wrapoperation.js

```diff
@@ -163,6 +163,21 @@ export default class WrapOperation extends Operation {
 		_move( wrappedRange, targetPosition );
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	toJSON() {
+		const json = super.toJSON();
+
+		json.position = this.position.toJSON();
+
+		if ( json.graveyardPosition ) {
```

Comment:
Wrap operation can have graveyard position or element. Shouldn't we do `json.element = this.element.toJSON()` ?